### ISSUE TITLE
select mirror

### ISSUE THREAD

**Issue by aeosynth** [voidlinux/void-mklive#103]
requesting a step to select repository mirror

**Comment by sholwe** [comment]
This would be useful- I did a live-to-install from the default distro, and was getting an abysmal 77kBs.

**Comment by the-maldridge** [comment]
This is possible to set by exporting XBPS_REPOSITORY before starting the installer, but I concur, it would be nice to have a menu choice for this.
